### PR TITLE
[don't merge] big change in readme

### PR DESCRIPTION
**Pull request by agramfort**
just a tiny test for a live demo

please do not merge this !

### CODE REVIEW

**Comment by dengemann** [comment]
@agramfort don't fix it if it's not  broken!!

File: README.rst [hunk 1]
```diff
@@ -39,6 +39,7 @@
 MNE-BIDS
 ========
 
+
```
Comment (dengemann) [inline README.rst]: remove this line!

Comment (agramfort) [reply README.rst]: you're right my bad...

File: README.rst [hunk 2]
```diff
@@ -39,6 +39,7 @@
...
 This is a repository for creating
```
Comment (agramfort) [inline README.rst]:
```suggestion
This is a repository for creating really cool
```

**Pull request closed without merging**